elide-model-config/src/main/resources/elideTableSchema.json

```diff
@@ -550,6 +550,27 @@
                             "dimensions"
                         ]
                     },
+                    {
+                        "properties": {
+                            "maker": {
+                                "title": "Table SQL maker function",
+                                "description": "JVM function to invoke to generate the SQL query which is used to populate the table.",
+                                "type": "string",
+                                "format": "javaClassName"
+                            },
+                            "dbConnectionName": {
+                                "title": "DB Connection Name",
+                                "description": "The database connection name for this model.",
+                                "type": "string",
+                                "format": "elideName"
+                            }
+                        },
+                        "required": [
```

Comment:
won't this mean we can not use FromSQL and Maker together?

Reply:
Correct.  It is one or the other.